src/build.py

```diff
@@ -875,6 +901,7 @@ def Emscripten(use_asm=True):
   def WriteEmscriptenConfig(infile, outfile):
     with open(infile) as config:
       text = config.read().replace('{{WASM_INSTALL}}', INSTALL_DIR)
```

Comment:
Not sure if it's reasonable to chain these like
```python
text = (config.read()
    .replace('{{WASM_INSTALL}}', INSTALL_DIR)
    .replace('{{PREBUILD_NODE}}', NODE_BIN)
    )
```

I prefer that because there's less line noise but it's pretty much equivalent. Maybe if we add another thing here?